Explorer/Assets/Scripts/ECS/Unity/GLTFContainer/Components/GltfContainerComponent.cs

```diff
@@ -22,5 +25,22 @@ public GltfContainerComponent(ColliderLayer visibleMeshesCollisionMask, Collider
             Promise = promise;
             State = LoadingState.Unknown;
         }
+
+        public static GltfContainerComponent CreateFaulty(string name)
+        {
+            GltfContainerComponent component = new GltfContainerComponent();
+            component.SetFaulty(name);
+            return component;
+        }
+            
+
+        public void SetFaulty(string name)
+        {
+            State = LoadingState.FinishedWithError;
+            Promise = AssetPromise<GltfContainerAsset, GetGltfContainerAssetIntention>.CreateFinalized(
```

Comment:
I believe it should set an exception instead unless FAULTY_HASH is somehow used. Especially considering you already create this exception. Thus, you replicate the previous behavior.